docs/workforce-management-framework/topics/pro-running-the-demo-repositories.adoc

```diff
@@ -1,91 +1,75 @@
 [id='{context}-pro-running-the-demo-repositories']
-= Running the {PROJECT_NAME} Applications on a Local Machine
+= Running the {PROJECT_NAME} Demonstration Environment
 
-The {PROJECT_NAME} {PROJECT_NAME} applications in this section are dependent on the _public NPM Registry_.
-
-This allows for a simpler setup of the {PROJECT_NAME} applications for users that only require the application to run, and will not engage in development.
+Get started by running a demonstration environment on your local machine. Follow these steps and use the _public NPM Registry_ to set up the {PROJECT_NAME} applications that contain sample Steps, Workflows, and Workorders.
 
 [discrete]
-== Prerequisites
-
-Ensure you have:
+== Requirements
 
-* Node and npm installed (tested on Node {WFM-RC-NodeVersion})
+The following must be installed on your system:
+* Node and npm (tested version: {WFM-RC-NodeVersion})
 * git
-* access to `git@github.com:feedhenry-raincatcher/raincatcher-server.git`
-* access to `git@github.com:feedhenry-raincatcher/raincatcher-mobile.git`
-* access to `git@github.com:feedhenry-raincatcher/raincatcher-portal.git`
-* MongoDB (tested with version: {WFM-RC-MongoDBVersion})
-* Redis (tested with version: {WFM-RC-RedisVersion})
+* MongoDB (tested version: {WFM-RC-MongoDBVersion})
```

Comment:
@austincunningham  I do not see this versions being populated.

Reply:
should I remove the tested versions?

Reply:
Actually that may be the best way to go.
Make sure to remove them from attributes.